docs/release-notes/highlights-7.2.0.asciidoc

```diff
@@ -6,11 +6,193 @@
 
 coming[7.2.0]
 
-See also <<breaking-changes>> and <<release-notes>>.
+Each release of {kib} brings new features and product improvements. 
+Here are the highlights of the feature and user experience changes in 7.2.0.
+
+Refer to the {kib} <<breaking-changes-7.2, breaking changes>> and <<release-notes, 
+release notes>> for a list of bug fixes and other changes.
 
 //NOTE: The notable-highlights tagged regions are re-used in the
 //Installation and Upgrade Guide
 
 // tag::notable-highlights[]
+// end::notable-highlights[]
+
+
+[float]
+=== Feature controls
+
+Want to hide Dev Tools from the left navigation? Show Stack Monitoring only to 
+admins? Or, give certain users access to only Dashboard and Canvas? 
+<<kibana-role-management, Feature controls>> allow you to hide and 
```

Comment:
FYI: Links like this will break if this section gets tagged for re-use in the Installation and Upgrade Guide in the 7.2 branch. i.e. they will need to be changed to fully-qualified links to the Kibana Guide.